docs/pages/kubernetes-access/guides/multiple-clusters.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Kubernetes Access Multiple Clusters
-description: Connecting Multiple Clusters with Teleport Kubernetes Access
+description: Connecting multiple clusters with Teleport Kubernetes access.
```

Comment:
How about we just name this Connecting a Kubernetes cluster to Teleport.

We can assume that it already supports multiple clusters, since this is the same pattern as server access.